packages/react-ui-components/src/SelectBox/complexOption.js

```diff
@@ -0,0 +1,35 @@
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import SelectBoxOption from './selectBoxOption';
+import style from './style.css';
+
+export default class ComplexOption extends PureComponent {
```

Comment:
would prefer if we find another name than "ComplexOption"; e.g. "MultilineOptionWithImage" -> but we can do this as a follow up